### PR TITLE
UX: close window on "escape" shortcut, refs #31

### PR DESCRIPTION
**Pull request by adnasa**
Closes window (on `escape`) and #31

__edit__: This was a lot easier than I thought 😂

### CODE REVIEW

File: app/main/index.js [hunk 1]
```diff
@@ -36,15 +36,23 @@ let win = null;
 // create a user config
 const config = new Config();
 
+const closeWindow = () => {
```
Comment (adnasa) [inline app/main/index.js]: We should consider modifying the eslint `no-use-before-define` rule to allow hoisting

File: app/main/index.js [hunk 2]
```diff
@@ -36,15 +36,23 @@ let win = null;
...
+  if (win.hide) {
+    win.hide();
+  }
+  return undefined;
```
Comment (adnasa) [inline app/main/index.js]: should be `return void 0`, but eslint was whining 😜

**Comment by vutran** [comment]
Hi thanks for the PR!

Can you update `hideWindow` to just return the expression itself? `eslint` is still complaining.

Other than that, looks good to me and keep up the good work!

**Comment by adnasa** [comment]
@vutran done 40b7739

**Comment by adnasa** [comment]
We should allow function hoisting as well (separate PR)

**Pull request merged**